doc/source/plugins/index.rst

```diff
@@ -113,8 +113,57 @@ B7xx     XSS
 Complete Test Plugin Listing
 ----------------------------
 
-.. toctree::
-   :maxdepth: 1
-   :glob:
-
-   *
+.. table:: Active Test Plugins
```

Comment:
This looks prone to merge conflict problems and seems to remove the error code from the specific plugin documentation page. 

If we don't care about merge conflicts running rampant, we can do 

```reSt
B101 :doc:`assert_unused <b101_assert_used>` Low High
```

To not show the error code in the name if we want without losing the error code on the plugin's page... unless we specifically want to start migrating away from the short code format

Reply:
I originally used the technique you describe, but the width of the lines became obscenely long. I also noticed that each plugin's doc already denotes the bandit ID anyway as their first line like so:
`B501: Test for missing certificate validation`

So it's already a bit redundant today in how it appears. For example:
https://bandit.readthedocs.io/en/1.7.4/plugins/b501_request_with_no_cert_validation.html